documentation/architecture/10_Data-Processing.md

```diff
@@ -62,11 +62,11 @@ The pre-processing module takes the raw data and transforms, joins and cleanses
 | **SEN** | sen.csv | URN | Contains information about the special educational characteristics of schools, including the number of pupils with a EHC Plan and with various other quantities that define the special education need charateristics of the school  | 
 | **Maintained School** | maintained_schools_master_list.csv | URN, UKPRN | This is the master list that represents the full cohort of LA maintained schools in the UK. This file also typically contains the financial data for the schools aswell. | 
 | **Academies** | academy_master_list.csv | Trust UPIN, Academy UPIN, URN | This is the master list that represents the full cohort of Academies in the UK. It is worth noting that this also contains entries for trusts |
-| **Academy Account Returns** | academy_ar.xlsx | Academy UPIN | This is the consolidated list of all Academy Account Returns financial data for the given financial year. | 
+| **Academy Account Returns** | ARX_CS_BenchmarkReport_YYYY.xlsx, ARX_BenchmarkReport_YYYY.xlsx, ARX_Standing_Data_YYYY.xlsx, ARX_Insights_Extract_YYYY.xlsx | Academy UPIN | This is the consolidated list of all Academy Account Returns financial data for the given financial year obtained from the AnM (RA_Datasets) database, where X is the version number (usually tied to the year) and YYYY is the year, e.g. 2023. Access to the AnM database can be requested via a "service now" ticket. These 4 files require aggregating and processing into the AAR report file supplied in the live service. | 
```

Comment:
Could the example include a full filename example? I was a little confused by the `X`.

Otherwise, this makes sense.

Reply:
I've expanded the example to include this